cmd/deploy_delete.go

```diff
@@ -23,8 +23,15 @@ func newDeleteDeploymentCmd(deployConfig *deployCommandConfig) *cobra.Command {
 	var deployConfigFile string
 	var deleteDeploymentCmd = &cobra.Command{
 		Use:   "delete",
-		Short: "Delete your deployed Appsody project from a Kubernetes cluster",
-		Long:  `This command deletes your deployed Appsody project from the configured Kubernetes cluster using your existing deployment manifest.`,
+		Short: "Delete your deployed Appsody project from a Kubernetes cluster.",
+		Long: `Delete your deployed Appsody project from the configured Kubernetes cluster using your existing deployment manifest.
```

Comment:
Needs a comma after "cluster", otherwise could be read as "the configured Kubernetes cluster that uses your existing deployment manifest":
"Delete your deployed Appsody project from the configured Kubernetes cluster, using your existing deployment manifest."